evm/scripts/deploy.ts

```diff
@@ -1,18 +1,12 @@
 import { ethers } from "hardhat";
 
 async function main() {
-  const currentTimestampInSeconds = Math.round(Date.now() / 1000);
-  const ONE_YEAR_IN_SECS = 365 * 24 * 60 * 60;
-  const unlockTime = currentTimestampInSeconds + ONE_YEAR_IN_SECS;
+  const MulitverseShip = await ethers.getContractFactory("Multiverse");
+  const ship = await MulitverseShip.deploy();
 
-  const lockedAmount = ethers.utils.parseEther("1");
+  await ship.deployed();
 
-  const Lock = await ethers.getContractFactory("Lock");
-  const lock = await Lock.deploy(unlockTime, { value: lockedAmount });
-
-  await lock.deployed();
-
-  console.log("Lock with 1 ETH deployed to:", lock.address);
+  console.log("Ship deployed to:", ship.address);
```

Comment:
Looks like we still need to have a task for recording the contract address once deployed?

Reply:
Yes, I'll be adding that to the next PR.